docusaurus/docs/extensions/extensions-configuration.md

```diff
@@ -89,7 +90,8 @@ ___`./pkg/my-package/package.json`___
       "catalog.cattle.io/rancher-version": ">= 2.7.7-0 < 2.9.0-0",
       "catalog.cattle.io/ui-extensions-version": ">= 1.1.0",
       "catalog.cattle.io/ui-version": ">= 2.7.7-0 < 2.9.0-0",
-      "catalog.cattle.io/display-name": "My Super Great Extension"
+      "catalog.cattle.io/display-name": "My Super Great Extension",
+      "noAuth": "true"
```

Comment:
noAuth is at the same level as annotations - it is not an annotation itself since it wires in differently - the value is a boolean, not a string

Reply:
Good catch.... I didn't notice that. `NO_AUTH=$(jq -r '.rancher.noAuth' < ${PACKAGE_JSON})`. You're right